ui/app/components/dashboard/learn-more-card.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import Component from '@glimmer/component';
+
+/**
+ * @module DashboardLearnMoreCard
+ * DashboardLearnMoreCard component are used to display external links
+ *
+ * @example
+ * ```js
+ * <DashboardLearnMoreCard  />
+ * ```
+ */
+
+export default class DashboardLearnMoreCard extends Component {
+  get learnMoreLinks() {
+    return [
+      {
+        link: 'https://developer.hashicorp.com/vault/tutorials/secrets-management',
+        icon: 'docs-link',
+        title: 'Secrets Management',
+      },
+      {
+        link: 'https://developer.hashicorp.com/vault/tutorials/monitoring',
+        icon: 'docs-link',
+        title: 'Monitor & Troubleshooting',
+      },
+      {
+        link: 'https://developer.hashicorp.com/vault/tutorials/adp/transform',
```

Comment:
Since this is an ADP feature, should it only show if the user has that feature?

Reply:
Good point! To clarify, is ADP transform? What's the best way to check for the capability of ADP users?